spec/sections/copies.bs

```diff
@@ -29,8 +29,16 @@ and "immediate" {{GPUQueue}} operations:
 - {{GPUQueue/copyExternalImageToTexture()}}, for copies from Web Platform image sources to textures
 
 Some texel values have <dfn dfn lt='equivalent texel representation'>multiple possible representations</dfn>
-of some values, e.g. as `r8snorm`, -1.0 can be represented as either -127 or -128.
-Copy commands are not guaranteed to preserve the source's bit-representation.
+of some values. As a result copy commands are not guaranteed to preserve the source's bit-representation.
+Examples include:
```

Comment:
The "multiple possible representations" wording doesn't quite make sense anymore. How about something like:
```
During a texel copy, a texel is copied over with an <dfn dfn>equivalent texel representation</dfn>:
Texel copies only guarantee that valid, normal numeric values in the source have the same numeric
value in the destination, not preserving other values, and not preserving the bit-representations:
```

Also I think this list is exhaustive so we don't have to say "examples include".

Reply:
Changed to language pretty close to this suggestion.